modules/ROOT/pages/syntax/parsing.adoc

```diff
@@ -0,0 +1,41 @@
+:description: This section describes whitespace in Cypher
+
+[[parsing]]
+= Parsing
+
+[abstract]
+--
+This section provides a general overview of how Cypher is parsing an input String, including whitespace.
+--
+
+Cypher considers input Strings as a sequence of Unicode UTF-16 code points. Unicodes can generally be escaped as `\uxxx`.
```

Comment:
I'm not that sure if they are utf-8 code points or utf-16 🤔. What makes you think they are utf-16?

Reply:
Hm....I stole this from GQL, but I agree it's not entirely clear to me, either. It looks like Java uses it's own representation.